gef-ui/server/utils/helpers.js

```diff
@@ -341,4 +544,7 @@ module.exports = {
   getUTCDate,
   coverDatesConfirmed,
   makerCanReSubmit,
+  hasChangedToIssued,
+  summaryItemsConditions,
+  facilityIssueDeadline,
```

Comment:
We need to split this file up; 550 lines of helper functions in one file is crazy 😢  Can we move the new functions into seperate files? Makes the tests easier to read as well

Reply:
I guess best way to split them is based on helpers related to:

1. Facilities
2. Deal
3. General

Any thoughts @ttbarnes ?

Reply:
Totally agree - although I wouldn't have a 'general' folder, just have general/generic helpers in the root helpers directory as their own functions. But then all other deal/facility helpers in their respective directories